nemo/nemo/core/neural_modules.py

```diff
@@ -404,19 +399,6 @@ def num_weights(self):
         """
         pass
 
-    @staticmethod
-    @abstractmethod
-    def create_ports(**kwargs):
-        """
```

Comment:
do not delete this method. 
Here should come deprecation warning saying how things should be done now. It also should keep behaving as it does now (but with deprecation warning). This will be in +1 release. In +2 release this method still should be there but will be throwing an exception saying how things should be done now

Reply:
I have already changed the documentation and explained how one should use those two methods instead of deprecated create_ports.

Of course we can leave create_ports (not abstract, but simply returning the deprecation warning), but please note that one still will have to implement _input_port_definitions()_ and _output_port_definitions()_, without them the module will be abstract - so what is really the point here, for me leaving that leaves a mess, not backward compatibility...

Reply:
I thought we were basically changing from static "create_ports" to member method "create_ports" (or member method "create_input_ports" and "create_output_ports").
That can be done with minimal, if any, breaking changes. Am I missing something?

Reply:
1) There are no braking changes.
2) I have always said that "create" is misleading, because we are not creating any physical instances of ports in those methods (btw. please note that you want those to be properties, not methods).